src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -170,4 +146,13 @@ public void addMessageWithoutPersistentStorage(Message message) {
   public void setMessages(List<Message> messages) {
     messages.forEach(message -> addMessageWithoutPersistentStorage(message));
   }
+  
+  private static void addToListInMap(map, key, value) {
+    if (map.containsKey(key)) {
```

Comment:
super nit: This can be further simplified to
if (!map.contains...) {
put array list
}
map.get.add...

You could argue that change would make it less readable though. Up to you.